index.html

```diff
@@ -315,20 +315,20 @@ <h2>
         </li>
         <li>If <a>obtaining the manifest</a> results in an error, the user
         agent MAY either:
-          <ol>
+          <ul>
             <li>Fall back to using the <a>top-level browsing context</a>
-            {{Document}}'s metadata to to populate <var>manifest</var> in a
+            {{Document}}'s metadata to populate <var>manifest</var> in a
             user-agent-specific way (e.g., setting
             |manifest|.{{WebAppManifest/name}} to the document <a data-cite=
             "HTML#the-title-element">`title`</a>) and considering the document
             <a>installable</a>.
             </li>
             <li>Or, consider the document not <a>installable</a>.
             </li>
-          </ol>
+          </ul>
         </li>
-        <li>Otherwise, the {{Document}} MAY be considered <a>installable</a>
-        (at the user agent's discretion; see [[[#installability-signals]]]).
+        <li>Otherwise, the {{Document}} is considered <a>installable</a> (at
```

Comment:
I don't know why "MAY" changed to "is". It's still a may, because you still allow user agent discretion.

Reply:
ah, good point... I'll flip the sentence around so that the MAY is applied to the user agent, not to the document.